packages/extensions-sdk/src/cli/commands/build.ts

```diff
@@ -100,7 +141,7 @@ function getRollupOptions(
 				styles(),
 				...plugins,
 				nodeResolve({ browser: true }),
-				commonjs({ esmExternals: true, sourceMap: false }),
+				commonjs({ esmExternals: true, sourceMap: true }),
```

Comment:
Curious to hear what advantages there are of enabling source map generation by the commonjs plugin.

Reply:
Oops that's committed by mistake.

I was looking to get sourcemaps of my modules to get a stacktrace when running a module within Directus. Without success.